playbook/package.json

```diff
@@ -60,6 +62,8 @@
     "svg-url-loader": "^8.0.0"
   },
   "devDependencies": {
+    "@actions/core": "1.10.0",
```

Comment:
Not sure why the github action stuff is populating in this PR. Might need to merge from master or something.

Reply:
Master is already merged. @jasperfurniss Do you want me to remove this line or something?